src/ansys/templates/python/solution/{{cookiecutter.__project_name_slug}}/pyproject.toml

```diff
@@ -44,8 +44,8 @@ priority = "primary"
 [tool.poetry.dependencies]
 python = ">=3.8, <3.11"
 ansys-dash-treeview = {version = "0.0.1.dev0", source = "solutions-private-pypi"}
-ansys-saf-glow = {version = "^0.2.0", source = "solutions-private-pypi" }
-ansys-saf-portal = {version = "^0.1.7", source = "solutions-private-pypi"}
+ansys-saf-glow = {version = "^0.3.dev4", source = "solutions-private-pypi" }
+ansys-saf-portal = {version = "^0.2.0", source = "solutions-private-pypi"}
```

Comment:
@ansJBoly @ansRMorin 
I have this [PR](https://github.com/ansys/ansys-templates/pull/274) to adjust the version, it uses 0.2.dev1 which includes the creation of the table when we start the solution. I think your PR is not needed.

Reply:
Okay, I'm going to close my PR.